flowmachine/flowmachine/core/server/query_schemas/base_exposed_query.py

```diff
@@ -88,35 +64,3 @@ def query_id(self):
         # by flowmachine.core.Query.md5, and the latter is currently being used by
         # the QueryStateMachine, so we need to use it to check the query state.
         return self._flowmachine_query_obj.md5
-
-    @property
-    def query_params(self):
```

Comment:
Fine with me to delete this (because we can't use it in its current form; see the comment in the `query_id` property above), but I'd be curious what your thinking is in general about this. Do you see a chance that in the future we may derive the query id purely from the query parameters (at least for exposed queries)? Or is this not possible/desirable?

Reply:
Well, if we want to continue to support flowmachine as a library, the method of deriving ids needs to be viable in both cases.